### PR TITLE
command: add demux-start-time property

### PR DESCRIPTION
**Pull request by a-tze**
I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: player/command.c [hunk 1]
```diff
@@ -1723,6 +1723,16 @@ static int mp_property_demuxer_cache_idle(void *ctx, struct m_property *prop,
     return m_property_flag_ro(action, arg, s.idle);
 }
 
+static int mp_property_demuxer_start_time(void *ctx, struct m_property *prop,
+                                          int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    if (!mpctx->playback_initialized)
+        return M_PROPERTY_UNAVAILABLE;
```
Comment (wm4) [inline player/command.c]: Checking whether mpctx->demuxer is set should be sufficient.

**Comment by wm4** [comment]
Manpage updates missing, also missing commit subject prefix.

**Comment by a-tze** [comment]
Should I put the update in input.rst only or also in interface-changes.rst?

**Comment by wm4** [comment]
Additions don't usually need to go to interface-changes.rst.

File: player/command.c (hunk 1 above)
Comment (wm4) [inline player/command.c]: Are you going to change this, or did you lose interest in this PR?

Comment (a-tze) [inline player/command.c]: Sorry, I missed that comment. Change incoming...

**Comment by a-tze** [comment]
The commits are squashed and rebased to current master.

**Comment by wm4** [comment]
Thanks, merged.

**Pull request closed without merging**